src/App/Pages/Settings/AutofillServicesPage.xaml.cs

```diff
@@ -55,11 +55,11 @@ private void ToggleInlineAutofill(object sender, EventArgs e)
             _vm.ToggleInlineAutofill();
         }
 
-        private void ToggleAccessibility(object sender, EventArgs e)
+        private async void ToggleAccessibilityAsync(object sender, EventArgs e)
         {
             if (DoOnce())
             {
-                _vm.ToggleAccessibility();
+                await _vm.ToggleAccessibilityAsync();
             }
         }
```

Comment:
This can be removed if we take the command approach. Furthermore, if we keep it like this it will crash the app if an exception is thrown.